### PR TITLE
Added a new plugin, RecoTauDiscriminantSimpleCut.cc, that applies the…

### PR DESCRIPTION
**Pull request by andrewj314**
… pT sum cut on isolation once, rather than for each WP. The config file HPSPFTaus_cff.py is also changed to invoke the new plugin.

### CODE REVIEW

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: RecoTauTag/RecoTau/plugins/RecoTauDiscriminantSimpleCut.cc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+#include "CommonTools/Utils/interface/StringObjectFunction.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "DataFormats/TauReco/interface/PFTauDiscriminator.h"
+#include "CondFormats/PhysicsToolsObjects/interface/PhysicsTGraphPayload.h"
+#include "CondFormats/DataRecord/interface/PhysicsTGraphPayloadRcd.h"
+#include "CondFormats/PhysicsToolsObjects/interface/PhysicsTFormulaPayload.h"
+#include "CondFormats/DataRecord/interface/PhysicsTFormulaPayloadRcd.h"
+
+#include "TMath.h"
```
Comment (slava77) [inline RecoTauTag/RecoTau/plugins/RecoTauDiscriminantSimpleCut.cc]: do you really need these ROOT includes here? 
Please remove

**Comment by slava77** [comment]
@andrewj314 
if I understood correctly, this change was not supported by PC in the meeting on Monday.
( @davidlange6 please confirm)
So, we should just go to 80X
Please make a PR for 80X.

**Comment by slava77** [comment]
-1

@andrewj314 
it looks like you have used the same topic branch for this PR  and for 80X.
Please close this 76X PR.
We should continue in #12261

**Pull request closed without merging**